central/networkgraph/service/service_impl_test.go

```diff
@@ -197,8 +197,8 @@ func (s *NetworkGraphServiceTestSuite) TestGetExternalNetworkFlows() {
 
 	mockFlowStore.EXPECT().GetMatchingFlows(gomock.Any(), gomock.Any(), gomock.Any()).Return(
 		[]*storage.NetworkFlow{
-			testutils.ExtFlow(es1aID.String(), "mydeployment"),
-			testutils.ExtFlow(es1bID.String(), "mydeployment"),
+			testutils.ExtFlowEgress(es1aID.String(), "mydeployment", "mycluster"),
+			testutils.ExtFlowEgress(es1bID.String(), "mydeployment", "mycluster"),
```

Comment:
`ExtFlow` is only used for testing. It was flawed, because it could not produce both valid ingress and egress flows. This was fixed by deleting it and creating `ExtFlowIngress` and `ExtFlowEgress`.